source/index.rst

```diff
@@ -89,7 +89,6 @@ These are institutions who were early adopters or provided HPC resources for dev
    applications
    user-documentation
 
-
```

Comment:
Note another unnecessary white space change. Now, these aren't a huge deal, but it's better not to change things that we don't need to. So how could this change be undone? One way to return a file to the state that it was in at a specific commit or tag is to check it out.

```sh
git checkout develop -- source/index.rst
```

In this case you do not need to make that change, but knowing how to get a specific version of a file in Git is useful.

Reply:
I will keep that in mind. Thanks.